microsoft-edge/extensions-chromium/publish/manage-settings.md

```diff
@@ -28,7 +28,11 @@ The **Contact info** section displays the details you provided when you created
   
 If your contact information changes edit the **Contact info** section.  Choose **Update**, make your changes, and then choose **Save**.  You may also choose **Contact info** in the left navigation menu to open the **Contact info** section.  
```

Comment:
If your contact information changes edit the **Contact info** section.  

To change your contact information, navigate to x> y> Contant Info section.